### ISSUE TITLE
generated example.html does not give protocol on script src

### ISSUE THREAD

**Issue by jowens** [ellisonbg/altair#177]
Maybe this is deliberate; the example.html generated on my install has the following:

```
  <script src="//d3js.org/d3.v3.min.js"></script>
  <script src="//vega.github.io/vega/vega.js"></script>
  <script src="//vega.github.io/vega-lite/vega-lite.js"></script>
  <script src="//vega.github.io/vega-editor/vendor/vega-embed.js" 
```

In Chrome, this produces the following:

![developer_tools_-_file____users_jowens_documents_working_gunrock-io_scripts_example_html](https://cloud.githubusercontent.com/assets/976970/17940900/f1339752-69e5-11e6-8d70-28f9fbb2abba.png)

When I add `http:` to the front of the src field, I can correctly view the visualization.

```
$ pip list | grep altair
altair (1.0.0)
```

**Comment by jowens** [comment]
I note https://github.com/ellisonbg/altair/commit/1088eba6981d27641059ae90793713f4fe848b3b addresses this (though it's not in the release, so maybe I need to stay on github trunk). Default of `local_file=True` is a little weird to me, but I trust y'all.

**Comment by jakevdp** [comment]
Yes, that commit is the relevant one. The PR with discussion is in #162. The issue is that if we explicitly add ``http`` then it doesn't work under ``https``, and vice-versa. The ``local`` options will make it into the next release some time this fall.

More broadly, we should note somewhere in the docs that the default template in ``to_html()`` is meant to be an example from which you could adapt your own desired HTML output, rather than meant to be a broad solution for all use-cases.

**Comment by jakevdp** [comment]
I'm going to close this because the issue is addressed by #162.

**Comment by jowens** [comment]
For posterity: I installed the git trunk version via pip with `sudo -H pip install git+git://github.com/ellisonbg/altair.git` and then used `open('example.html', 'w').write(chart.to_html(local_file=False))` successfully.
